src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1054,6 +1092,12 @@ class ImGuiConsole
 		ImVec2 contentSize = ImGui::GetContentRegionAvail();
 		contentSize.y -= footer_height_to_reserve;
 
+		// Set Font for ZepEditor
+		if (s_updateConsoleFont)
+		{
+			m_zepEditor->SetFont(Zep::ZepTextType::Text, ImGui::GetFont(), s_consoleFontSize);
```

Comment:
the console specifically uses the console font. This gets the default font. Theres a big mismatch here in the logic.